docs/source/developer/virtualdom.create.tsx

```diff
@@ -0,0 +1,6 @@
+import * as React from 'react';
+
+import { Widget } from '@phosphor/widgets';
+import { ReactWidget } from '@jupyterlab/apputils';
+
+const myWidget: Widget = ReactWidget.create(<div>My Widget</div>);
```

Comment:
So far I've been using a pattern like this. I find it a little more accessible/comparable to my vanilla React experience, as long as it's not a pattern we want to discourage due to particularities of the codebase. It's more verbose than the current example but more extensible.

If the next example is left at `<div>My Widget</div>` as a shorthand for creating a `ReactRenderElement`, then both styles are demonstrated.

```{javascript}
class MyReactComponent extends React.Component {
  render() {
    return (
      <div>Hello!</div>
    );
  }
}

const myWidget: Widget = ReactWidget.create(<MyReactComponent />);
```

Reply:
Yeah that looks good. What do you think about showing a functional component so it's like:

```typescript
function MyComponent() {
  return <div>My Widget</div>;
}

const myWidget: Widget = ReactWidget.create(<MyComponent />);
```

Reply:
Yeah that also looks good. And it's still brief.